src/main/steps/applicant1/hub-page/right-column/template.njk

```diff
@@ -8,6 +8,9 @@
 {% if hasCertificateOfService %}
   <p class="govuk-body">{{ certificateOfServiceDownloadLink | safe }}</p>
 {% endif %}
+{% if hasCertificateOfDeemedOrDispensedService %}
+  <p class="govuk-body">{{ deemedOrDispensedDownloadLink | safe }}</p>
```

Comment:
why not move the anchor tag here, and pass the href and download info through the content